pkg/txt/datetime_filepath_test.go

```diff
@@ -61,13 +61,13 @@ func TestDateFromFilePath(t *testing.T) {
 	})
 	t.Run("/2020/1212/20130518_142022_3D657EBD.jpg", func(t *testing.T) {
 		result := DateFromFilePath("/2020/1212/20130518_142022_3D657EBD.jpg")
-		//assert.False(t, result.IsZero())
-		assert.True(t, result.IsZero())
+		assert.False(t, result.IsZero())
+		assert.Equal(t, "2013-05-18 00:00:00 +0000 UTC", result.String())
 	})
 	t.Run("20130518_142022_3D657EBD.jpg", func(t *testing.T) {
 		result := DateFromFilePath("20130518_142022_3D657EBD.jpg")
-		//assert.False(t, result.IsZero())
-		assert.True(t, result.IsZero())
+		assert.False(t, result.IsZero())
```

Comment:
That must not work as you'll otherwise end up with "known" dates derived from filenames automatically created during import, although the original filename did NOT contain any such date. This is why import filenames are explicitly excluded right now.

Reply:
Ah, I see. Didn't knew about the filename that is generated by PhotoPiew itself.

So for WhatsApp images the filter would have to explicitly only allow IMG-YYYYMMDD and VID-YYYYMMDD. Is this a good idea or does it just get too specific